setup.py

```diff
@@ -71,16 +71,18 @@
     'future==0.15.2',
     'futures==3.0.5',
     'google-cloud==0.19.0',
+    'google-api-python-client==1.6.0',
+    'seaborn==0.7.1',
+    'plotly==1.12.12',
     'httplib2==0.9.2',
     'oauth2client==2.2.0',
+    'psutil==5.0.1',
     'pandas>=0.17.1',
     'pandas-profiling>=1.0.0a2',
     'python-dateutil==2.5.0',
     'pytz>=2015.4',
     'pyyaml==3.11',
     'requests==2.9.1',
-    'scipy==0.18.0',
```

Comment:
These two are not directly used by our code, but I think that we've included them because we want any of our users to be able to use them out of the box.

@chmeyers and @qimingj might know better about that, though.